frontend/src/components/Sidebar/SidebarData.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Link } from "react-router";
+import { Link } from "react-router-dom";
```

Comment:
@bunny-mack I tried really hard to keep your file, but this is the only way it worked